posthog/api/feature_flag.py

```diff
@@ -100,7 +100,25 @@ def has_object_permission(self, request: Request, view, feature_flag) -> bool:
         if request.method in SAFE_METHODS:
             return True
         else:
-            return can_user_edit_feature_flag(request, feature_flag)
+            return (
```

Comment:
The double checks were being used in the serializer but not the core permission.